dexbot/strategies/base.py

```diff
@@ -153,9 +153,9 @@ def configure(cls, return_base_config=True):
             ConfigElement('account', 'string', '', 'Account',
                           'BitShares account name for the bot to operate with',
                           ''),
-            ConfigElement('market', 'string', 'USD:BTS', 'Market',
-                          'BitShares market to operate on, in the format ASSET:OTHERASSET, for example \"USD:BTS\"',
-                          r'[A-Z0-9\.]+[:\/][A-Z0-9\.]+'),
+            ConfigElement('market', 'string', 'USD/BTS', 'Market',
```

Comment:
This assumes USD is QUOTE which is confusing, please change to BTS/USD.

Reply:
done.